task01-html-css/characters.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<!-- 
+  All Rights Reserved
+  This software is property information of AKUREY S.A.
+  Contact us at contact@akurey.com
+  @summary short description for the file
+-->
+<link rel="stylesheet" href="characters.css">
+<link href='https://fonts.googleapis.com/css?family=Roboto Slab' rel='stylesheet'>
+<body>
+    <div class="grid-container">
+        <div class="grid-item">
+            <div class="filter-item"></div>
+            <img class="image-item" src="./images/Rectangle 4.png" alt="Monkey">
+            <div class="text-item" style="left: 25.34px;">Monkey D. Luffy</div>
```

Comment:
Delete the 'left' styling everywhere, and avoid hardcoding an specific style for every single element, it's not a good practice